File: The Core/metroCard.cs
int[] metroCard(int lastNumberOfDays) {
    switch (lastNumberOfDays){
        case 28:
        case 30:
            return new int[]{31};
        case 31:
            return new int[]{28, 30, 31};
        default:
            return null;
    }
}

File: The Core/At The Crossroads/16. Metro Card/metroCard.lua
function metroCard(lastNumberOfDays)
    if lastNumberOfDays == 28 then 
        return {31}
    elseif lastNumberOfDays == 30 then 
        return {31}
    elseif lastNumberOfDays == 31 then 
        return {28, 30, 31}
    else 
        return null
    end
end
